setup/src/Magento/Setup/Console/Command/ModuleConfigStatusCommand.php

```diff
@@ -0,0 +1,97 @@
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Setup\Console\Command;
+
+use Magento\Framework\App\DeploymentConfig\Reader as ConfigReader;
+use Magento\Framework\Config\ConfigOptionsListConstants;
+use Magento\Framework\Config\File\ConfigFilePool;
+use Magento\Framework\Setup\ConsoleLogger;
+use Magento\Setup\Model\InstallerFactory;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Command to check if the modules config in app/etc/config.php matches with how Magento interprets it
+ */
+class ModuleConfigStatusCommand extends Command
+{
+    /**
+     * Deployment config reader
+     *
+     * @var ConfigReader
+     */
+    private $configReader;
+
+    /**
+     * Installer service factory.
+     *
+     * @var InstallerFactory
+     */
+    private $installerFactory;
+
+    /**
+     * @param ConfigReader     $configReader
+     * @param InstallerFactory $installerFactory
+     */
+    public function __construct(
+        ConfigReader     $configReader,
+        InstallerFactory $installerFactory
+    ) {
+        $this->configReader     = $configReader;
+        $this->installerFactory = $installerFactory;
+
+        parent::__construct();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure()
+    {
+        $this->setName('module:config:status')
+             ->setDescription('Checks the modules in the app/etc/config.php file, to see if they are correct');
+
+        parent::configure();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            // the config as currently in app/etc/config.php
+            $currentConfig = $this->configReader->load(ConfigFilePool::APP_CONFIG);
+            if (!array_key_exists(ConfigOptionsListConstants::KEY_MODULES, $currentConfig)) {
+                throw new \Exception('Can\'t find the modules config in the \'app/etc/config.php\' file.');
+            }
+
+            $currentModuleConfig = $currentConfig[ConfigOptionsListConstants::KEY_MODULES];
+
+            $installer = $this->installerFactory->create(new ConsoleLogger($output));
+
+            // the module config as Magento calculated it
+            $correctModuleConfig = $installer->getModulesConfig();
+
+            if ($currentModuleConfig !== $correctModuleConfig) {
+                throw new \Exception(
+                    'The current module config in the \'app/etc/config.php\' file is incorrect. Run \'setup:upgrade\' to fix it.'
+                );
+            }
+
+            $output->writeln(
+                '<info>Everything seems to be in order.</info>'
+            );
+        } catch (\Exception $e) {
+            $output->writeln('<error>' . $e->getMessage() . '</error>');
+
+            return \Magento\Framework\Console\Cli::RETURN_FAILURE;
```

Comment:
Please move the `Magento\Framework\Console\Cli` class name to the imports as well.
```
use Magento\Framework\Console\Cli;
```
```
return Cli::RETURN_FAILURE;
```